src/task.rs

```diff
@@ -1453,6 +1499,11 @@ impl TaskMonitor {
         self.metrics.slow_poll_threshold
     }
 
+    /// The threshold after which a scheduling delay will be considered "long".
```

Comment:
Can you follow the documentation pattern of [`slow_poll_threshold`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMonitor.html#method.slow_poll_threshold) here?